tests/components/media_source/test_models.py

```diff
@@ -62,6 +64,7 @@ async def test_browse_media_parent_no_children():
     assert not item["can_play"]
     assert item["can_expand"]
     assert len(item["children"]) == 0
+    assert item["children_media_class"] is None
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6158285)